types/src/on_chain_config/execution_config.rs

```diff
@@ -72,14 +72,15 @@ impl OnChainExecutionConfig {
         OnChainExecutionConfig::V4(ExecutionConfigV4 {
             transaction_shuffler_type: TransactionShufflerType::SenderAwareV2(32),
             block_gas_limit_type: BlockGasLimitType::ComplexLimitV1 {
-                effective_block_gas_limit: 35000,
+                effective_block_gas_limit: 40000,
                 execution_gas_effective_multiplier: 1,
                 io_gas_effective_multiplier: 1,
-                block_output_limit: None,
-                conflict_penalty_window: 1,
+                conflict_penalty_window: 6,
+                use_granular_resource_group_conflicts: false,
+                use_module_publishing_block_conflict: false,
```

Comment:
I assume we're not on V4 anywhere? And we'll push it out with 1.9?

Reply:
yes!